client/components/email-verification/verify-email-dialog.jsx

```diff
@@ -75,15 +78,17 @@ class VerifyEmailDialog extends React.Component {
 		const strings = {
 			confirmHeading: i18n.translate( 'Please confirm your email address' ),
 
-			confirmExplanation: i18n.translate( 'We sent you an email when you first signed up. Please open the message and click the blue button.' ),
+			confirmExplanation: i18n.translate( 'We sent you an email when you first signed up.' +
+				'Please open the message and click the blue button.' ),
 
-			confirmReasoning: i18n.translate( 'Email confirmation allows us to assist when recovering your account in the event you forget your password.' ),
+			confirmReasoning: i18n.translate( 'Email confirmation allows us to assist when recovering' +
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/12596/commits/03ec2c77a0b8303bb69224b06262e59fe6a6b9d0